### PR TITLE
Versioning update

### PR DESCRIPTION
**Pull request by Alexsandruss**
Changes proposed in this pull request:
- Avoid double copy of static libs on Win
- Store version in one place (makefile.ver)
- Example and samples link to non-versioned Win library file

### CODE REVIEW

**Comment by Alexsandruss** [comment]
http://intel-ci.intel.com/eb29d248-aa67-f15f-9f85-8cdcd4b723a1

File: deploy/local/vars_fbsd.sh [hunk 1]
```diff
@@ -88,8 +88,8 @@ if [ ! -d $__daal_tmp_dir ]; then
     __daal_tmp_dir=$(dirname -- "$(get_script_path "$vars_script_name")")
 fi
 
-export DAL_MAJOR_BINARY=1
-export DAL_MINOR_BINARY=0
+export DAL_MAJOR_BINARY=__DAL_MAJOR_BINARY__
+export DAL_MINOR_BINARY=__DAL_MINOR_BINARY__
```
Comment (napetrov) [inline deploy/local/vars_fbsd.sh:92]: for master it should be also bumped to 1.1

Comment (Alexsandruss) [reply deploy/local/vars_fbsd.sh:92]: Fixed, minor version was increased

**Comment by Alexsandruss** [comment]
http://intel-ci.intel.com/eb2b1892-3925-f13a-b398-8cdcd4b723a1

**Comment by Alexsandruss** [comment]
http://intel-ci.intel.com/eb2d7b23-61a0-f18b-b3f6-8cdcd4b723a1

**Review by PetrovKP — APPROVED** [review]

**Comment by Alexsandruss** [comment]
@napetrov @rlnx @SmirnovEgorRu, do you have any comments?

**Comment by napetrov** [comment]
There are failures in module-files check


88c88
< DAL_MINOR_BINARY=0
---
> DAL_MINOR_BINARY=1

**Comment by Alexsandruss** [comment]
> There are failures in module-files check
> 
> ## 88c88
> < DAL_MINOR_BINARY=0
> > DAL_MINOR_BINARY=1

Fixed by excluding of DAL_MINOR_BINARY and DAL_MAJOR_BINARY in check: http://intel-ci.intel.com/eb2f06bf-1781-f1f6-a458-8cdcd4b723a1

**Comment by Alexsandruss** [comment]
http://intel-ci.intel.com/eb2f1098-0ccd-f18f-bafb-8cdcd4b723a1

**Review by napetrov — APPROVED** [review]

**Pull request merged**